.github/workflows/ci-cd-infra.yml

```
@@ -21,12 +21,18 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v4
 
+      - name: Skip pipeline trigger when AWS role is unavailable
+        if: ${{ secrets.AWS_CI_CD_INFRA_ROLE_TO_ASSUME_TEST == '' }}
```

actionlint (1.7.11)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)